tests/Fixtures/ColourTest.php

```diff
@@ -6,20 +6,26 @@
 use Fixtures\Innmind\Colour\Colour;
 use Innmind\Colour\RGBA;
 use Innmind\BlackBox\Set;
+use Innmind\BlackBox\Random;
 use PHPUnit\Framework\TestCase;
 
-class ColourTest extends TestCase
+class ColourTest extends TestCase implements Random
 {
     public function testInterface()
     {
         $set = Colour::any();
 
         $this->assertInstanceOf(Set::class, $set);
 
-        foreach ($set->values() as $value) {
+        foreach ($set->values(new self) as $value) {
```

Comment:
Could you please use `Innmind\BlckBox\Random\RandomInt` instead ? The test class shouldn't implement a behaviour